### PR TITLE
lua integration

### LINKED ISSUE #13: Integrate lua interpreter

### PR DESCRIPTION
**Pull request by BaderBC**
Closes #13
- lua integrated
- api.Package fixed

### CODE REVIEW

File: src/lua/api_binding.go
Comment (maxidragon) [inline src/lua/api_binding.go:9]: there is a typo 😉
```suggestion
// Every api needs to be attached here in order to be available:
```

File: src/lua/interpreter.go
Comment (cooligus) [inline src/lua/interpreter.go:18]:
```go
return false, err
```

**Review by cooligus — CHANGES_REQUESTED** [review]
Don't forget to add GetDaemon

**Review by cooligus — APPROVED** [review]

**Review by maxidragon — APPROVED** [review]
Please wait for @cooligus approval before merging

**Pull request merged**
